app/views/pages/home.html.haml

```diff
@@ -1,7 +1,39 @@
 .container
   .col-md-8.col-md-offset-2.text-xs-center
-    %h1 Lorem Ipsum is simply dummy text.
-    %h5 Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
+    -# %form.form-inline
+    -#   .form-group
```

Comment:
Codebase needs to be clean with just the code we need. If we need this commented out code later, we can get it back from git history.